addons/sale/models/sales_team.py

```diff
@@ -3,41 +3,57 @@
 
 from datetime import date
 
-from odoo import api, fields, models
+from odoo import api, fields, models, _
 
 
 class CrmTeam(models.Model):
     _inherit = 'crm.team'
 
-    use_quotations = fields.Boolean('Quotations', default=True,
-        help="Check this box to manage quotations in this sales team.")
-    use_invoices = fields.Boolean('Invoices',
-        help="Check this box to manage invoices in this sales team.")
+    use_quotations = fields.Boolean(
+        string='Quotations', default=True,
+        help="Check this box to manage quotations in this sales channel.")
+    use_invoices = fields.Boolean('Invoicing Target', help="Check this box to set an invoicing target for this sales channel.")
     invoiced = fields.Integer(
         compute='_compute_invoiced',
         string='Invoiced This Month', readonly=True,
         help="Invoice revenue for the current month. This is the amount the sales "
-                "team has invoiced this month. It is used to compute the progression ratio "
+                "channel has invoiced this month. It is used to compute the progression ratio "
                 "of the current and target revenue on the kanban view.")
     invoiced_target = fields.Integer(
         string='Invoice Target',
         help="Target of invoice revenue for the current month. This is the amount the sales "
-             "team estimates to be able to invoice this month.")
-    sales_to_invoice_amount = fields.Integer(
-        compute='_compute_sales_to_invoice_amount',
-        string='Amount of sales to invoice', readonly=True,
-    )
-    currency_id = fields.Many2one("res.currency", related='company_id.currency_id',
-        string="Currency", readonly=True, required=True)
+             "channel estimates to be able to invoice this month.")
+    quotations_to_invoice = fields.Integer(
+        compute='_compute_quotations_to_invoice',
+        string='Number of quotations to invoice', readonly=True)
+    quotations_to_invoice_amount = fields.Integer(
+        compute='_compute_quotations_to_invoice',
+        string='Amount of quotations to invoice', readonly=True)
+    sales_to_invoice = fields.Integer(
+        compute='_compute_sales_to_invoice',
+        string='Number of sales to invoice', readonly=True)
+    dashboard_graph_model = fields.Selection(selection_add=[
+        ('sales', 'Sales'),
+        ('invoices', 'Invoices'),
+    ], default='sales')
+
+    def _compute_quotations_to_invoice(self):
+        amounts = self.env['sale.report'].read_group([
+            ('team_id', 'in', self.ids),
+            ('state', 'in', ['draft', 'sent']),
+        ], ['price_total', 'team_id', 'name'], ['team_id', 'name'], lazy=False)
+        for rec in amounts:
+            self.browse(rec['team_id'][0]).quotations_to_invoice_amount += rec['price_total']
```

Comment:
Sadly they will remain disturbed. As we need to group by team_id and by name.
sale.report is reporting on sale.order.lines, so we need to group them by their reference (=name) too. This makes the extra (needless) data formatting too big in my opinion.